tests/xtro-sharpie/macOS-StoreKit.todo

```diff
@@ -0,0 +1 @@
+!missing-selector! SKProduct::isDownloadable not bound
```

Comment:
should be in .ignore (there's nothing we can do) with a comment/link to the trello/rdar